packages/experience-builder-types/src/types.ts

```diff
@@ -243,6 +243,7 @@ export type ExperienceComponentSettings = {
 export type Composition = {
   title: string;
   slug: string;
+  tokenDefinition?: DesignTokensDefinition;
```

Comment:
This is type for the fields of the Experience Entry (previously called composition). `tokenDefinitions` won't be a separate field, i.e. defined in the Content Type, right? So I don't think we need it here